src/ol/render/webgl/PolygonReplay.js

```diff
@@ -873,14 +873,14 @@ WebGLPolygonReplay.prototype.getDeleteResourcesFunction = function(context) {
  */
 WebGLPolygonReplay.prototype.setUpProgram = function(gl, context, size, pixelRatio) {
   // get the program
-  const fragmentShader = _ol_render_webgl_polygonreplay_defaultshader_.fragment;
-  const vertexShader = _ol_render_webgl_polygonreplay_defaultshader_.vertex;
+  const fragmentShader = fragment;
+  const vertexShader = vertex;
   const program = context.getProgram(fragmentShader, vertexShader);
```

Comment:
It looks like the two declarations above could be removed and this could become:
```js
const program = context.getProgram(fragment, vertex);
```

Same in other modules.

Reply:
done in 2b6a3aa